system/HTTP/Files/FileCollection.php

```diff
@@ -92,26 +92,16 @@ public function getFile(string $name)
 			{
 				$name         = explode('.', $name);
 				$uploadedFile = $this->getValueDotNotationSyntax($name, $this->files);
-				if ($uploadedFile instanceof \CodeIgniter\HTTP\Files\UploadedFile)
-				{
-					return $uploadedFile;
-				}
-
-				return null;
+				return ($uploadedFile instanceof \CodeIgniter\HTTP\Files\UploadedFile) ?
```

Comment:
in same namespace, use direct `instanceof UploadedFile` should be enough:

```suggestion
                              return ($uploadedFile instanceof UploadedFile) ?
```

Reply:
Good suggestion. I don't know why it was fully qualified before.